packages/airnode-examples/integrations/relay-security-schemes/config.example.json

```diff
@@ -15,9 +15,9 @@
       "type": "evm",
       "options": {
         "txType": "eip1559",
-        "baseFeeMultiplier": "2",
+        "baseFeeMultiplier": 2,
         "priorityFee": {
-          "value": "3.12",
```

Comment:
What's the advantage of changing this? We could lose precision by using JS numbers?

Reply:
Tbh, I'm not sure besides perhaps keeping things more consistent with other config fields (see this [task](https://api3dao.atlassian.net/browse/AN-489))

Reply:
I originally made them strings when I implemented eip1559 pricing, I think because of the imprecision issue and I think the value was being ingested underneath by ethers as a string but I didn't feel strongly either way. There was an extended discussion on the topic and the outcome was that they be changed to `number`. Discussion here: https://api3workspace.slack.com/archives/C02AYRX8D89/p1641981419038200?thread_ts=1641981419.038200&cid=C02AYRX8D89

Reply:
Sorry I missed that thread. I still think strings would be slightly safer but don't feel strongly enough about it. As you mentioned, these numbers should always be really small and not require much precision.

Reply:
Yeah it's a bit of an imaginary problem